packages/core/src/App/Containers/RealAccountSignup/real-account-signup.jsx

```diff
@@ -277,6 +278,33 @@ const RealAccountSignup = ({
 
     const [assessment_decline, setAssessmentDecline] = React.useState(false);
 
+    const analytic_data = React.useMemo(
+        () => ({
+            form_source: document.referrer,
+            form_name: 'real_account_signup_form',
+            landing_company: real_account_signup_target,
+        }),
+        [real_account_signup_target]
+    );
```

Comment:
```suggestion
    const analytic_data = React.useMemo(
        () => ({
            form_source: document.referrer,
            form_name: 'real_account_signup_form',
            landing_company: real_account_signup_target,
        }),
        [real_account_signup_target]
    );
```
@shahzaib-deriv, since `document.referrer` is not a reactive value, don't you think it'd be better to compute this object inside inside of the  `trackEvent` function itself, since that's the only place it's being used, and `real_account_signup_target` is already a dependancy of that function's `useCallback`. This would help to get rid of the `useMemo`. What's your opinion on this? 🤔

Reply:
Thanks for noticing @jim-deriv. Makes sense to move it together. Will check & update 👍

Reply:
Hi @jim-deriv actually for tracking purpose we no need to change this value at all. For CRO it'd be nice to have at this referrer at least once for event
In global scope it'll be hard to explain this detail for everyone, but in the same time in object it looks good. Again, for CRO squad needs